src/main/java/org/dependencytrack/resources/v1/ProjectResource.java

```diff
@@ -741,7 +741,7 @@ public Response patchProject(
                 modified |= setIfDifferent(jsonProject, project, Project::getCpe, Project::setCpe);
                 modified |= setIfDifferent(jsonProject, project, Project::getPurl, Project::setPurl);
                 modified |= setIfDifferent(jsonProject, project, Project::getSwidTagId, Project::setSwidTagId);
-                modified |= setIfDifferent(jsonProject, project, Project::isActive, Project::setActive);
+                modified |= setIfDifferent(jsonProject, project, Project::getInactiveSince, Project::setInactiveSince);
```

Comment:
This allows clients to choose a custom date, which I think is not what we want. The client should be telling us that a project shall be deactivated or reactivated, but not *when* it was deactivated. The latter is an implementation detail for us, that should not "bleed" through the API.